### PR TITLE
Add management workload annotations

### PR DESCRIPTION
**Pull request by lack**
In support of the workload partitioning feature
(https://github.com/openshift/enhancements/pull/703), we need to add
annotations to all management pods and namespaces so they can be
properly identified and assigned to segregated management cores on
clusters configured to do so.

### Description

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA:
- Enhancement proposal:

### CODE REVIEW

File: olm_deploy/operatorregistry/registry-deployment.yaml
Comment (jcantrill) [inline olm_deploy/operatorregistry/registry-deployment.yaml:13]: This deployment is only used during testing.  Do we still need the change here?

Comment (lack) [reply olm_deploy/operatorregistry/registry-deployment.yaml:13]: No, in that case it probably doesn't matter.  I was perhaps overly exuberant in my grepping :)

**Review by jcantrill — COMMENTED** [review]
/approve

@vimalk78  please :lgtm after you determine if there are any changes needed for this to make it into our CPaaS builds

File: olm_deploy/operatorregistry/registry-deployment.yaml
Comment (lack) [reply olm_deploy/operatorregistry/registry-deployment.yaml:13]: Would you like me to remove this one?

**Comment by lack** [comment]
/hold

There may be one further update to this PR yet.

**Comment by imiller0** [comment]
/unhold
openshift/kubernetes#632 has now merged.

**Comment by lack** [comment]
@vimalk78 : Ready for final review now!

**Comment by vimalk78** [comment]
there are no changes required from cpaas builds point of view
/lgtm

**Review by vimalk78 — APPROVED** [review]

**Review by igor-karpukhin — APPROVED** [review]

**Pull request merged**
